packages/service-provider-core/src/watch-options.ts

```diff
@@ -0,0 +1,14 @@
+export default interface WatchOptions {
+  /** Allowed values: ‘default’, ‘updateLookup’. When set to ‘updateLookup’, the change stream will include both a delta describing the changes to the document, as well as a copy of the entire document that was changed from some time after the change occurred. */
+  fullDocument?: 'default' | 'updateLookup' | undefined;
+  /** The maximum amount of time for the server to wait on new documents to satisfy a change stream query. */
+  maxAwaitTimeMS?: number;
+  /** Allows you to start a changeStream after a specified event. See {@link https://docs.mongodb.com/master/changeStreams/#resumeafter-for-change-streams|ChangeStream documentation}. */
+  resumeAfter?: any; // TODO: see MONGOSH-427
```

Comment:
These, as far as I can tell, have to be `any` since they are expecting the typescript type from the node driver, which we can't import in this package.